docs/core/unmanaged-api/metadata/interfaces/imetadataconverter-getmetadatafromtypeinfo-method.md

```diff
@@ -2,52 +2,47 @@
 description: "Learn more about: IMetaDataConverter::GetMetaDataFromTypeInfo Method"
 title: "IMetaDataConverter::GetMetaDataFromTypeInfo Method"
 ms.date: "03/30/2017"
-api_name: 
+api_name:
   - "IMetaDataConverter.GetMetaDataFromTypeInfo"
-api_location: 
+api_location:
   - "mscoree.dll"
-api_type: 
+api_type:
   - "COM"
-f1_keywords: 
+f1_keywords:
   - "IMetaDataConverter::GetMetaDataFromTypeInfo"
-helpviewer_keywords: 
-  - "GetMetaDataFromTypeInfo method [.NET Framework metadata]"
   - "IMetaDataConverter::GetMetaDataFromTypeInfo method [.NET Framework metadata]"
-ms.assetid: d44484bb-23a3-49c3-9e46-69d0d9ab4f0f
-topic_type: 
+topic_type:
   - "apiref"
 ---
 # IMetaDataConverter::GetMetaDataFromTypeInfo Method
```

Comment:
IMetadataConverter does not exist in .NET Core. It should stay in the .NET Framework specific part.